tests/views/test_classification_type_view.py

```diff
@@ -4,24 +4,35 @@
 class TestClassificationTypeView(TestClient):
 
     def _create_classification_type(self, classification_type):
-        response = self.client.post('/classificationtype/{}'.format(classification_type))
+        response = self.client.post('/classificationtype/{}'.format(classification_type),
+                                    headers=self.get_auth_headers())
         self.assertStatus(response, 201)
 
     def test_upload_classification_type(self):
         # when we upload a new classification type
         classification_type = "LEGAL_BASIS"
-        response = self.client.post('/classificationtype/{}'.format(classification_type))
+        response = self.client.post('/classificationtype/{}'.format(classification_type),
+                                    headers=self.get_auth_headers())
 
         # Then the classification type is successfully uploaded to the database
         self.assertStatus(response, 201)
 
+    def test_create_classification_type_without_basic_auth(self):
+        # when we upload a new classification type
+        classification_type = "LEGAL_BASIS"
+        response = self.client.post('/classificationtype/{}'.format(classification_type))
```

Comment:
Could use `f'string interpolation'` for consistency

Reply:
Thought I'd got them all

Reply:
They sneak up on you ;)